templates/todo/grafbase/schema.graphql

```diff
@@ -1,12 +1,12 @@
 type TodoList @model {
   id: ID!
-  title: String!
+  title: String! @unique
```

Comment:
Why restrict the title to be unique? I have multiple todo lists with the same name in my Notes app